geometry_msgs/PointStamped location
geometry_msgs/Vector3Stamped speed
int32 id
string identity
time firstSeen
time lastSeen
int32 specialFlag
